config: mainnet genesis update

txhsl opened the pull request:
Include #279 and some other governance setting modifications.

txhsl committed:
config: add Multisig system contract byte code to mainnet genesis cfg

txhsl committed:
config: remove unused zero paddings

AnnaShaleva commented on config/genesis_mainnet.json:
```diff
@@ -54,8 +54,8 @@
         "0x0000000000000000000000000000000000000000000000000000000000000002": "0x0de0b6b3a7640000",
-        "0x0000000000000000000000000000000000000000000000000000000000000003": "0x027b46536c66c8e3000000",
-        "0x0000000000000000000000000000000000000000000000000000000000000004": "0x043c33c1937564800000",
+        "0x0000000000000000000000000000000000000000000000000000000000000003": "0x04f68ca6d8cd91c0000000",
```
Are you sure that this value is 6e24? Because I'm getting another value, try this Go program:
```
package main

import (
	"fmt"
	"math/big"
)

func main() {
	actual, ok := new(big.Int).SetString("04f68ca6d8cd91c0000000", 16)
	if !ok {
		fmt.Println(ok)
		return
	}
	fmt.Println(actual.String())
	fmt.Println(actual.Text(16))

	expected := big.NewInt(1000000000000000000)
	expected.Mul(expected, big.NewInt(6000000))
	fmt.Println(expected.String())
	fmt.Println(expected.Text(16))
}
```
The output of this program is:
```
5999999999999999899336704
4f68ca6d8cd91c0000000
6000000000000000000000000
4f68ca6d8cd91c6000000
```
Although I agree that `expected`'s hex representation may be missing the leading zeroes.

txhsl replied:
Yeh, `4f68ca6d8cd91c0000000` is inaccurate.

AnnaShaleva reviewed (commented):
Also please, adjust the documentation accordingly.

txhsl committed:
config: update MainNet register fee and vote target amount settings

Set Governance candidate registration fee to 1000 GAS and target vote
amount to 6000000 GAS. Also adjust the system contracts documentation
to include an updated MainNet Governance settings.

txhsl commented:
> Also please, adjust the documentation accordingly.

Done.

AnnaShaleva reviewed (commented):
Please, just rebase onto `prepare-rel-v0.2.1` branch (I've just rebased `prepare-rel-v0.2.1` onto fresh bane-main), no need to merge changes from bane-main. Also, squash all logical blocks into commits, it will be easier for us to track changes and to write changelog.

txhsl commented:
@AnnaShaleva I wish this rebase merge works as you expected.

AnnaShaleva commented:
Let me check one more time.

AnnaShaleva commented:
@txhsl, please check the updated version. I did `git rebase origin/prepare-rel-v0.2.1 and then squashed two latest commits of yours. As a result we have three nice and clean commits.

txhsl commented:
> @txhsl, please check the updated version. I did `git rebase origin/prepare-rel-v0.2.1 and then squashed two latest commits of yours. As a result we have three nice and clean commits.

Thanks.
